### PR TITLE
PUBDEV-2785

### PR DESCRIPTION
**Pull request by angela0xdata**
- Added tests to pybooklet.python.vignette.py file for each example.
- Updated PythonBooklet.tex file
- Updated python/ipython_dataprep_output_fixed.py. This file now only
includes examples that we do not want to run. (It includes a lot sample
and output text rather than actual values.)
- Removed other unnecessary items in the /python folder.

---
This change is [Reviewable](https://reviewable.io/reviews/h2oai/h2o-3/146)

### CODE REVIEW

File: h2o-docs/src/booklets/v2_2015/source/PythonBooklet.tex [hunk 1]
```diff
@@ -73,7 +73,7 @@
 
 Machine Learning with Python and H2O\\
   by Spencer Aiello, Cliff Click, \\ Hank Roark \&\  Ludi Rehak\\
```
Comment (arnocandel) [inline h2o-docs/src/booklets/v2_2015/source/PythonBooklet.tex]: Should probably add Pasha here as well.

File: h2o-docs/src/booklets/v2_2015/source/PythonBooklet.tex [hunk 2]
```diff
@@ -171,7 +173,7 @@ \subsection{Installation in Python}
 
 To load a recent H2O package from PyPI, run:
 
-\begin{lstlisting}[style=python]
+\begin{lstlisting}[style=pythoncode]
```
Comment (st-pasha) [inline h2o-docs/src/booklets/v2_2015/source/PythonBooklet.tex]: This is bash, not python. You run this from a command line

**Comment by angela0xdata** [comment]
I changed that line to normal syntax. In the PDF output, the line number no longer appears with the code.

**Comment by st-pasha** [comment]
1) Section {Citation} does not match the front page of the booklet
2) Section {5.4.1 Pipelines}: line `h2o.__PROGRESS_BAR__=False` is a noop and should be removed.

Other than that it seems good.

**Comment by angela0xdata** [comment]
Fixed the citation and removed that PROGRESS_BAR line.

**Pull request merged**